CMakeLists.txt

```diff
@@ -230,8 +231,14 @@ if(UNIX)
     set(AVIF_PLATFORM_LIBRARIES m Threads::Threads)
 endif()
 
-find_package(libyuv QUIET) # not required
+find_package(libyuv) # not required
```

Comment:
The line
```-- Found libyuv: path/to/libyuv.a```
Is useful to indicate which library file is being linked against, so I removed the `QUIET`. Inside `Findlibyuv.cmake` the call `pkg_check_modules(_LIBYUV libyuv QUIET)` doesn't provide extra info, so I silented that one.